app/models/base_email_settings.rb

```diff
@@ -26,4 +28,25 @@ def params(setting_name)
   def service_email_from
     from_address.email_address
   end
+
+  def reference_number_enabled?
+    ServiceConfiguration.find_by(service_id: service.service_id, deployment_environment: deployment_environment, name: 'REFERENCE_NUMBER').present?
+  end
+
+  def insert_reference_number_sentence(content)
+    reference_number_sentence = I18n.t('default_values.reference_number_sentence')
+    content.gsub(REFERENCE_NUMBER_PLACEHOLDER, reference_number_sentence)
+  end
+
+  def remove_reference_number_sentence(content)
+    content.gsub(REFERENCE_NUMBER_PLACEHOLDER, '')
+  end
+
+  def update_default_value(setting)
```

Comment:
You updated this as I was in the middle of suggesting to refactor to having a method instead of all the ternarys!

My only thought now is that `update_default_value()` is not overly descriptive, and maybe too generic - I don't intrinsically understand what it is going to do, and also we're likely to need another method like this for payment links.

My initial thought had been `substitute_reference_number(content)`
but thinking ahead maybe `substitute_placeholder(content, placeholder)` is better?

We would then end up with something like:
```
def confirmation_email_body
  substitute_placeholder(settings_for(:confirmation_email_body), REFERENCE_NUMBER_PLACEHOLDER)
  substitute_placeholder(settings_for(:confirmation_email_body), PAYMENT_LINK_PLACEHOLDER)
end
```